action.yml

```diff
@@ -23,11 +27,17 @@ runs:
   using: 'composite'
   steps:
     - run: |
+        response=$(curl --fail-with-body --silent --show-error \
+          --request POST \
+          --header "Authorization: Bearer ${{ env.SLACK_BOT_TOKEN || inputs.token }}" \
+          --header "Content-Type: application/json; charset=utf-8" \
+          --url https://slack.com/api/chat.postMessage \
+          --data "{\"channel\": \"${{ inputs.channel }}\", \"thread_ts\": \"${{ inputs.thread_ts }}\", \"unfurl_links\": ${{ inputs.unfurl_links }}, \"text\": \"${{ inputs.text }}\"}" \
+        )
```

Comment:
My goals with these changes are:
- make this step fail if we get a response with HTTP status >= 400 (i.e., didn't work)
- capture the response and log it (to make debugging easier if anything _does_ go wrong)
- set the `ts` output for this action by processing the curl response

But I'm a total bash/curl newb, so please feel free to make suggestions! 🙏